packages/common/src/ui/layout/tables/RemoteDataTable.tsx

```diff
@@ -7,6 +7,7 @@ import {
   useRowSelect,
   useSortBy,
   useTable,
+  useFlexLayout,
```

Comment:
In the end all this really does is pass the width as a flex basis to the cells 🤔